### PR TITLE
Spavlov/d add documentation about threaded comments

### PR DESCRIPTION
**Pull request by Svilen-Pavlov**
Added article with screenshots.

### CODE REVIEW

File: controls/radspreadsheet/features/comments.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+page_title: Threaded Comments 
+slug: radspreadsheet-features-comments
+tags: thread, comments, xlsx, 
+published: True
+position: 13
+---
+
+# Comments
+
+Since R1 2022 the Spreadsheet control supports working with comments.
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/comments.md:12]: R2 2022

File: controls/radspreadsheet/features/comments.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+![RadSpreadProcessing Comments](images/RadSpreadProcessing_Comments_02.png)
+
+#### Figure 3: Add Comment from the Comments group
+
+![RadSpreadProcessing Comments](images/RadSpreadProcessing_Comments_03.png)
+
+As you can see in the above image, the Comments group allows you to perform the following actions:
+
+* __New Comment:__ Adds new comment to the selected cell.
+* __Reply Comment:__ Adds a reply to an existing comment to the selected cell. (only available when selected cell already contains a comment)
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/comments.md:35]: * __Reply Comment:__ Adds a reply to an existing comment. This option is only available when the selected cell already contains a comment.

File: controls/radspreadsheet/features/comments.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* __Delete:__ Deletes the comment in the selected cell, including all of its replies. (only available when selected cell already contains a comment)
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/comments.md:36]: * __Delete:__ Deletes the comment in the selected cell, including all of its replies. Only available when the selected cell already contains a comment.

File: controls/radspreadsheet/features/comments.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+* __Previous Comment:__ Selects the previous Comment in the current spreadsheet.
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/comments.md:37]: Does that work for the whole document or for the active worksheet only? If it is for the worksheet, change the statement above

**Comment by Svilen-Pavlov** [comment]
Done, Merging is blocked for me though.

File: controls/radspreadsheet/features/comments.md (hunk 4 above)
Comment (TanyaDimitrova) [reply controls/radspreadsheet/features/comments.md]: Spreadsheet means the whole document, change the term to Worksheet

**Review by TanyaDimitrova — APPROVED** [review]

**Pull request merged**